__tests__/index.js

```diff
@@ -395,22 +395,37 @@ describe('Manipulate', () => {
 
 describe('Display', () => {
   it('Format', () => {
-    const m = moment(time).format('dddd, MMMM D YYYY, h:mm:ss A');
-    const d = date.format(new Date(time), 'eeee, MMMM d yyyy, h:mm:ss aa', {
+    const m = moment(time).format('dddd, MMMM D, YYYY, h:mm:ss A');
+    const n = new Date(time).toLocaleDateString(undefined, {
+      weekday: 'long',
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+      hour: 'numeric',
+      minute: 'numeric',
+      second: 'numeric',
+    });
+    const d = date.format(new Date(time), 'eeee, MMMM d, yyyy, h:mm:ss aa', {
       awareOfUnicodeTokens: true,
     });
-    const day = dayjs(time).format('dddd, MMMM D YYYY, h:mm:ss A');
+    const day = dayjs(time).format('dddd, MMMM D, YYYY, h:mm:ss A');
     const l = DateTime.fromMillis(time).toFormat(
-      'EEEE, MMMM d yyyy, h:mm:ss a'
+      'EEEE, MMMM d, yyyy, h:mm:ss a'
     );
+    expect(m).toBe(n);
```

Comment:
This test failed. Native `new Date` does not support am/pm as far as i know :)

Reply:
Passed/passes for me (and in Travis?), but does adding `hour12: true` in the object do it for you?